x-pack/plugins/ml/public/application/components/model_snapshots/revert_model_snapshot_flyout/create_calendar.tsx

```diff
@@ -0,0 +1,289 @@
+    }
+  }
+
+  return (
+    <>
+      <EuiSpacer size="l" />
+      <div>
+        <FormattedMessage
+          id="xpack.ml.revertModelSnapshotFlyout.createCalendar.title"
+          defaultMessage="Select time range for calendar event."
+        />
+      </div>
+      <EuiSpacer size="m" />
+      <Chart
+        eventRateData={eventRateData}
+        anomalies={anomalies}
+        loading={chartReady === false}
+        overlayRanges={calendarEvents.filter(filterIncompleteEvents).map((c) => ({
+          start: c.start!.valueOf(),
+          end: c.end!.valueOf(),
+        }))}
+        onBrushEnd={onBrushEnd}
+      />
+      <EuiSpacer size="s" />
+
+      {calendarEvents.map((c, i) => (
+        <div key={i}>
+          <EuiPanel paddingSize="s">
+            <EuiFlexGroup>
+              <EuiFlexItem>
+                <EuiFlexGroup>
+                  <EuiFlexItem>
+                    <EuiFormRow
+                      label={i18n.translate(
+                        'xpack.ml.revertModelSnapshotFlyout.createCalendar.fromLabel',
+                        {
+                          defaultMessage: 'From',
+                        }
+                      )}
+                    >
+                      <EuiDatePicker
+                        showTimeSelect
+                        selected={c.start}
+                        minDate={minSelectableTimeMoment}
+                        maxDate={c.end ?? maxSelectableTimeMoment}
+                        onChange={(d) => setStartDate(d, i)}
+                      />
+                    </EuiFormRow>
+                  </EuiFlexItem>
+                  <EuiFlexItem>
+                    <EuiFormRow
+                      label={i18n.translate(
+                        'xpack.ml.revertModelSnapshotFlyout.createCalendar.toLabel',
+                        {
+                          defaultMessage: 'To',
+                        }
+                      )}
+                    >
+                      <EuiDatePicker
+                        showTimeSelect
+                        selected={c.end}
+                        minDate={c.start ?? minSelectableTimeMoment}
+                        maxDate={maxSelectableTimeMoment}
+                        onChange={(d) => setEndDate(d, i)}
+                      />
+                    </EuiFormRow>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+                <EuiSpacer size="s" />
+                <EuiFlexGroup>
+                  <EuiFlexItem>
+                    <EuiFormRow
+                      fullWidth
+                      label={i18n.translate(
+                        'xpack.ml.revertModelSnapshotFlyout.createCalendar.descriptionLabel',
+                        {
+                          defaultMessage: 'Description',
+                        }
+                      )}
+                    >
+                      <EuiFieldText
+                        style={{ width: '100%', maxWidth: 'inherit' }}
```

Comment:
you can use a `fullWidth` prop

Reply:
Changed in ac22c8ce86162fd9d4db4687850de4ae33873f4b